packages/datepicker/datepicker.scss

```diff
@@ -109,7 +109,7 @@ $bottom-margin: $component-spacing--large;
         position: relative;
         flex-grow: 2;
 
-        & > .jkl-dropdown__value {
+        & > .jkl-select__value {
```

Comment:
I think we should create our own css class and not reuse the jkl-select__value.

Reply:
Good catch!